staff_features/accessions/step_definitions/accession_spawn_digital_object.rb

```diff
@@ -0,0 +1,40 @@
+# frozen_string_literal: true
+
+When 'the {string} setting is enabled in the Repository Preferences' do |repository_setting_checkbox_label|
+  find('#user-menu-dropdown').click
+  click_on 'Repository Preferences (admin)'
+
+  check repository_setting_checkbox_label
+
+  click_on 'Save'
+end
+
+Then 'the Create Digital Object modal is displayed' do
+  expect(page).to have_css '#accession_instances__0__digital_object__ref__modal'
+end
+
+Then 'the Digital Object title is filled in with the Accession Title' do
+  expect(find('#digital_object_title_').value).to eq "Accession Title #{@uuid}"
+end
+
+Then 'the following Digital Object forms have the same values as the Accession' do |linked_record_forms|
+  linked_record_forms.raw.each do |form_title|
+    form_title = form_title[0]
+
+    section_title = find('h3', text: form_title)
+    section = section_title.ancestor('section')
+    expect(section[:id]).to_not eq nil
+
+    case form_title
+    when 'Languages'
+      expect(find('#digital_object_lang_materials__0__language_and_script__language_').value).to eq 'English, Middle (1100-1500)'
```

Comment:
This expectation fails on my local machine consistently, even though it passed on CI. The step definition 'an Accession has been created' that creates this value does so by entering "english" in the dropdown and selecting the first item. When I manually access the server in my e2e-tests container running locally and enter "english" in the language dropdown, "English, Old (ca.450-1100)" is the first item that appears, not "English, Middle (1100-1500)" as expected here. Further, in the records that the step definition created when I ran the tests, it ended up selecting the plain "English" item. It seems that the dropdown suggestion behavior (or the search results it uses, anyway) is not always consistent in its ordering. Just to be safe, it may be a good idea to update that step definition in accession_shared.rb to enter the exact text instead of just a substring that has multiple matches. (And, probably to keep this in mind for similar situations where a dropdown search is performed.)

Reply:
Fixed.